CONTRIBUTING.md

```diff
@@ -2,6 +2,30 @@
 
 Contributions welcome - just raise a PR and make sure the tests pass!
 
+## Testing
+
+**First make sure you have the following dependencies.**
+
+- [rbenv](https://github.com/rbenv/rbenv#installation)
+
+**Next install Ruby and the gems required for testing.**
+
+```sh
+rbenv install
+gem install bundler
+bundle install
+```
+
+**Now use one of the following commands to run tests.**
```

Comment:
Do we need to give people both options? could leave the `bundle exec rspec` equivalent as an advanced step that people can find themselves.

Reply:
That's fair - someone can look at the Makefile to understand the whole test suite. I've reworded this section.